protocol-dashboard/src/services/Audius/sdk.ts

```diff
@@ -51,15 +39,8 @@ const initAudiusSdk = ({
       })
     }
   })
-  sdkResolve()
-  isReady = true
 }
 
-const audiusSdk = async () => {
-  if (!isReady) {
-    await sdkPromise
-  }
-  return instance
-}
+initAudiusSdk()
```

Comment:
we don't need to init sdk in `setup.ts` anymore since we don't need the auth functions, so we can just init sdk here and get rid of the async stuff.

Reply:
Nice